warehouse/accounts/views.py

```diff
@@ -215,6 +216,14 @@ def register(request, _form_class=RegistrationForm):
     if request.authenticated_userid is not None:
         return HTTPSeeOther("/")
 
+    if AdminFlag().is_enabled(request.db, 'disallow-new-user-registration'):
```

Comment:
You don't need to instantiate `AdminFlag` here, should just be `AdminFlag.is_enabled(...)`. Same with the other callsite.

Reply:
d'oh missed that in factoring to a `classmethod`. thanks.